app/javascript/utils/form/final_form/Field.js

```diff
@@ -89,6 +89,7 @@ export const CurrencyField = ({ fieldName, validation, id, label, placeholder, m
           id={id || `form-${fieldName}`}
           fieldName={fieldName} />
         <input {...input}
+          type='text'
```

Comment:
What does adding this line do? Is this what triggers the border radius?

Reply:
yes! the border radius styling depends on having the input type defined